pool/upgrades.go

```diff
@@ -15,16 +17,26 @@ const (
 	// transactionId field to the payments struct for payment tracking purposes.
 	transactionIDVersion = 1
 
+	// shareIDVersion is the second version of the database. It updates
+	// the share id key and removes the created on time field.
+	shareIDVersion = 2
```

Comment:
Both upgrades are being included in a single PR, so why do we need to increment the db version twice? Will every individual DB change always correspond to a new db version?

Reply:
For db upgrades it is ideal to keep them focus on a single change to make it easier to test against the change made and track bugs if there is an issue with an upgrade. We are not combining upgrades for this reason. The db versions here are tied to a single upgrade here, so yes every db upgrade will correspond to a new db version.